anypoint-mq/4.x/modules/ROOT/pages/anypoint-mq-consume.adoc

```diff
@@ -1,6 +1,15 @@
 = Anypoint MQ Consume Operation - Mule 4
 
-The `consume` operation in the Anypoint MQ connector enables the app consume a single message at any time of the flow, from any destination, using the acknowledgment strategy that supports your use case.
+The Consume operation in the Anypoint MQ connector enables the app consume a single message from a queue at any time of the flow, from any destination, using the acknowledgment strategy that supports your use case.
```

Comment:
1. enables the app "to" consume
2. from a queue and from any destination are referring to the same thing in this case right? can we get rid of one of them?

Reply:
Yes!